mheimbinder/AirCastingiOS
# Build incrementation fix

For now, the base version if somewhere = 1 and somewhere = 2 which result in push Build #2 to Firebase 3rd time.

Maybe it can be resolved better, but I know this is working. Did my best for now

To test:
run `fastlane run increment_build_number` and see the results before and after use of this PR
